src/logic/operate.js

```diff
@@ -0,0 +1,30 @@
+import Big from 'big';
+
+const calc = (() => {
+  const operate = (numberOne, numberTwo, operator) => {
+    let total;
+    const first = Big(numberOne);
+    const second = Big(numberTwo);
+
+    if (operator === '+') {
+      total = first + second;
+    } else if (operator === '-') {
+      total = first - second;
+    } else if (operator === 'X') {
+      total = first * second;
+    } else if (operator === '/') {
+      total = first / second;
+    } else if (operator === '+') {
+      total = first + second;
+    } else if (operator === '%' && numberTwo === null) {
+      total = first / 100;
+    } else if (operator === '%' && numberTwo !== null) {
+      total = (first / 100) * second;
+    }
```

Comment:
For the calculations done here, the +,-,*,/,% operator all have functions in the Big.js package module, please use them to perform calculations
example. num1.div(num2)  would perform the division operation, see big.js documentation for more